docs/sources/developers/plugins/working-with-data-frames.md

```diff
@@ -0,0 +1,109 @@
++++
+title = "Working with data frames"
+type = "docs"
++++
+
+# Working with data frames
+
+The data frame is a columnar data structure, which allows efficient querying of large amounts of data. Data sources create data frames for panels to visualize. Since data frames are a central concept when developing plugins for Grafana, in this guide we'll look at some ways you can use them.
+
+The [DataFrame]({{< relref "../../packages_api/data/dataframe.md" >}}>) interface contains a `name` and an array of `fields` where each field contains the name, type, and the values for the field.
```

Comment:
The title of the linked document is "Data frames." I'm not sure what "DataFrame" is referring to here.

Reply:
DataFrame refers to the reference documentation for that interface. The reference seems right to me, it points to https://grafana.com/docs/grafana/latest/packages_api/data/dataframe/

Reply:
Silly me, I thought this linked to the Data frames doc. Disregard!